test/fixtures/basic/server/api/very-long-request.ts

```diff
@@ -0,0 +1,7 @@
+export default defineEventHandler(async () => {
+  await timeout(1000)
```

Comment:
this probably could be much shorter for the sake of our CI

Reply:
```suggestion
  await timeout(20)
```

Reply:
what do you think of 60ms ? (i forgot to set this pr to draft 😅 )

Reply:
perfect. oops, sorry for the quick review